lib/modules/datasource/unity3d/index.ts

```diff
@@ -24,6 +24,10 @@ export class Unity3dDatasource extends Datasource {
 
   override readonly registryStrategy = 'merge';
 
+  override readonly releaseTimestampSupport = true;
+  override readonly releaseTimeStampNote =
```

Comment:
Timestamp is the common term, it doesn't need to be written with separate words

```suggestion
  override readonly releaseTimestampNote =
```